interoperator/pkg/internal/resources/resources.go

```diff
@@ -491,26 +492,33 @@ func (r resourceManager) deleteSubResource(client kubernetes.Client, resource *u
 				Name:      resource.GetName(),
 				Namespace: resource.GetNamespace(),
 			}
-			err := client.Get(context.TODO(), namespacedName, resource)
-			if err != nil {
-				return err
-			}
-			content := resource.UnstructuredContent()
-			statusInt, ok := content["status"]
-			var status map[string]interface{}
-			if ok {
-				status, ok = statusInt.(map[string]interface{})
-				if !ok {
-					return fmt.Errorf("status field not map for resource %v", resource)
+
+			err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
```

Comment:
Wouldn't it retry every-time an error occurs ? Also the `DefaultRetry` seems to be 5 times.

Is it possible to do it only for the specific error type ?

Reply:
Retry happens only on Conflict error